inc/admin.php

```diff
@@ -70,14 +71,16 @@ function nkdeletepending_options_page() {
 					*/
 					global $wpdb;
 					$wpdb->query( 
-						$wpdb->prepare( "DELETE FROM $wpdb->comments WHERE comment_approved = 0" )
+						$wpdb->prepare( "DELETE FROM $wpdb->comments WHERE comment_approved = %d", 0 )
```

Comment:
This change is not required. We need to use `prepare` only if we are passing custom variables to the query.

Reply:
Fixed.